arch/xtensa/src/esp32/esp32_tim.c

```diff
@@ -252,13 +248,10 @@ static void esp32_tim_modifyreg32(FAR struct esp32_tim_dev_s *dev,
  *
  ****************************************************************************/
 
-static int esp32_tim_start(FAR struct esp32_tim_dev_s *dev)
+static void esp32_tim_start(FAR struct esp32_tim_dev_s *dev)
```

Comment:
Why to change it to void? The timer API is expecting a value:
```
    case TCIOC_START:
      {
        /* Start the timer, resetting the time to the current timeout */

        if (lower->ops->start)
          {
            ret = lower->ops->start(lower);
          }
```

Reply:
Please, note that this start is from lower half layer not from `esp32_tim.c`. The start function from lower half is still returning int value.

Reply:
> Please, note that this start is from lower half layer not from `esp32_tim.c`. The start function from lower half is still returning int value.

Yes, I noticed it later. But I think it is a good idea to follow the same standard. Also to keep in sync with the other timer drivers.